hudi-spark/src/main/java/org/apache/hudi/keygen/ComplexKeyGenerator.java

```diff
@@ -49,11 +49,11 @@
 
   public ComplexKeyGenerator(TypedProperties props) {
     super(props);
-    this.recordKeyFields = Arrays.asList(props.getString(DataSourceWriteOptions.RECORDKEY_FIELD_OPT_KEY()).split(","))
-            .stream().map(String::trim).collect(Collectors.toList());
+    this.recordKeyFields = Arrays.stream(props.getString(DataSourceWriteOptions.RECORDKEY_FIELD_OPT_KEY())
```

Comment:
These changes belong to the field of code cleanup. Based on the rule that one PR does one thing. It would be better to refactor it in a separate PR **next time**.

Reply:
sure